zng/type.go

```diff
@@ -275,8 +277,6 @@ func LookupPrimitiveById(id int) Type {
 		return TypeTime
 	case IdDuration:
 		return TypeDuration
-	case IdType:
```

Comment:
Delete the IdType const as well?

Reply:
Ruh, roh.  We need to keep IdType because type values are stored as primitives in zng but managed as complex types in resolver.Context at runtime...  it's a bit of a hybrid approach but I feel it is the right design.  I will get this working soon and push an update.

Reply:
Ok, I changed my mind about using type context for the type/zson hash table.  I created package zson (which is where I put the zson parser in a branch under dev) and put the hash table in zson.